tutorcodejail/templates/codejail/build/codejail/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM docker.io/ubuntu:20.04 as minimal
+MAINTAINER Overhang.io <contact@overhang.io>
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt update && \
+    apt install -y build-essential curl git language-pack-en
+ENV LC_ALL en_US.UTF-8
+
+###### Install python with pyenv in /opt/pyenv and create virtualenv in /openedx/venv
+FROM minimal as python
+# https://github.com/pyenv/pyenv/wiki/Common-build-problems#prerequisites
+RUN apt update && \
+    apt install -y libssl-dev zlib1g-dev libbz2-dev \
+    libreadline-dev libsqlite3-dev wget curl llvm libncurses5-dev libncursesw5-dev \
+    xz-utils tk-dev libffi-dev liblzma-dev python-openssl git subversion
+ENV PYENV_ROOT /opt/pyenv
+RUN git clone https://github.com/pyenv/pyenv $PYENV_ROOT --branch v1.2.21 --depth 1
+
+ARG CODEJAILSERVICE_PYTHON_VERSION=3.8.6
+RUN $PYENV_ROOT/bin/pyenv install $CODEJAILSERVICE_PYTHON_VERSION
+
+ARG SANDBOX_PYTHON_VERSION=3.5.10
+RUN $PYENV_ROOT/bin/pyenv install $SANDBOX_PYTHON_VERSION
+
+RUN $PYENV_ROOT/versions/$CODEJAILSERVICE_PYTHON_VERSION/bin/python -m venv /openedx/venv
+RUN $PYENV_ROOT/versions/$SANDBOX_PYTHON_VERSION/bin/python -m venv --copies /sandbox/venv
+
+###### Codejail service code
+FROM minimal as code
+RUN git clone https://github.com/eduNEXT/codejailservice.git --branch eric/mvp --depth 1 /openedx/codejailservice
+WORKDIR /openedx/codejailservice
+
+###### Install python requirements in virtualenv
+FROM python as codejailservice-python-requirements
+
+ENV PATH /openedx/venv/bin:${PATH}
+ENV VIRTUAL_ENV /openedx/venv/
+
+COPY --from=code /openedx/codejailservice /openedx/codejailservice
+WORKDIR /openedx/codejailservice
+RUN pip3 install -r requirements/base.txt
+RUN pip3 install ipdb
+
+###### Install python requirements in virtualenv
+FROM python as sandbox-python-requirements
+
+ENV PATH /sandbox/venv/bin:${PATH}
+ENV VIRTUAL_ENV /sandbox/venv/
+
+WORKDIR /var/tmp
+RUN mkdir -p common/lib/
+RUN svn export https://github.com/edx/edx-platform.git/tags/open-release/koa.3/common/lib/sandbox-packages common/lib/sandbox-packages
+RUN svn export https://github.com/edx/edx-platform.git/tags/open-release/koa.3/common/lib/symmath common/lib/symmath
```

Comment:
What does `svn export` do? Do we really need subversion or can we get rid of it? If we just to copy a file from edx-platform, it would be safer to `COPY` it from the docker image labeled `DOCKER_IMAGE_OPENEDX`.

Reply:
You are right, it is a good idea to copy it from the edx-platform image, I am going to try that.